tests/library/defaultbrowsercontext-2.spec.ts

```diff
@@ -101,6 +101,20 @@ it('should accept userDataDir', async ({ createUserDataDir, browserType }) => {
   expect(fs.readdirSync(userDataDir).length).toBeGreaterThan(0);
 });
 
+it('should accept relative userDataDir', async ({ createUserDataDir, browserType }) => {
+  const userDataDir = await createUserDataDir();
+  const cwd = process.cwd();
+  try {
+    console.log(userDataDir);
+    process.chdir(userDataDir);
```

Comment:
Instead of this, perhaps pass `path.relative(process.cwd(), path.join(userDataDir, 'foobar'))`?

Reply:
Not a huge of polluting the actual cwd during the test run hence doing it this way.